content/guides/agent_checks.md

```diff
@@ -476,26 +476,33 @@ Troubleshooting
 ======================================================
 -->
 
-<h3 id="troubleshooting">Troubleshooting</h3>
+### Troubleshooting
 
 Custom Agent checks can't be directly called from python and instead
  need to be called by the agent. To test this, run:
 
-    sudo -u dd-agent dd-agent check my_check
+    sudo -u dd-agent dd-agent check <CHECK_NAME>
 
 If your issue continues, please reach out to Support with the help page that
  lists the paths it installs.
 
-<h4>Testing Custom Checks on Windows</h4>
+#### Testing custom checks on Windows
 
+* **For agent version < 5.12**:<br>
 Testing custom checks on Windows is easy. The Agent install includes a file called shell.exe
```

Comment:
Let's remove "Testing custom checks on Windows is easy." Not only is it unnecessary to say, saying it in this bullet point could imply that it's NOT easy with Agent > 5.12.